tests/test_datadog_utils.py

```diff
@@ -50,6 +50,20 @@ def test_default_json_payload_values_are_correct(self, mock_post):
         payload = json.loads(kwargs['json'])
         self.assertEqual(payload['series'][0]['metric'], 'name')
         self.assertEqual(payload['series'][0]['points'], [[1577874030, 'value']])
+
+    def test_host_and_environment_is_unknown_by_default(self, mock_post):
+        self._setup_env(DATADOG_API_KEY='api_key', DATADOG_APP_KEY='app_key')
+        # workaround for running this test within github actions
+        if os.environ.get(GITHUB_ACTIONS) == 'true':
+            def reset_github_action():
+                os.environ[GITHUB_ACTIONS] = 'true'
+            os.environ[GITHUB_ACTIONS] = 'false'
+            self.addCleanup(reset_github_action)
```

Comment:
this makes me die a little bit inside, but it was what came to mind for working around this issue.

The reason the code that checks for the `GITHUB_ACTIONS` env var exists is because it uses that info to set the environment and host values on the metric it sends to datadog. However now I'm wondering if we should actually reserve that for the "environment" the code runs in ("HQ", "Vellum", etc). That way we could continue to use the metric name virtually as is (only specifies language), and organize dashboards based on the environment value?

Reply:
> That way we could continue to use the metric name virtually as is (only specifies language), and organize dashboards based on the environment value?

Yes, that seems right to me.

Reply:
So would you agree that there isn't a need to include where the metric info was collected from? Meaning, from a github action vs from a manual run or something like that? I don't see us taking advantage of that info in any meaningful way at the moment, so tempted to just omit it entirely.

Reply:
Offlined and since `host` is an expected metric value when using datadog's API, I'm going to remove the `environment` tag and rely on `host` to represent which project/repository the metrics were run on (e.g., hq, vellum, etc)

Reply:
See 5561aaa5b34f1038cda2753849b7c1fee2ee7a39 and 30405988a8cecb882131e96bbef3a3591ca2d7b7